From bbfab2bfccea5edec7fe83cf940231b099254288 Mon Sep 17 00:00:00 2001
From: Avinash Kumar <avikumar@habana.ai>
Date: Tue, 12 Oct 2021 20:09:12 +0300
Subject: [PATCH] [SW-58555] pid: enable boot_info module

ARC PID reads boot_info  HL_BOOT_INFO_FW_CFG_SKIP status.
For this zephy module boot_info needs to be added to
arc pid.

Change-Id: Icfcaf6d034d8ed00756ebf52acfce8bfd593113b
Signed-off-by: Avinash Kumar <avikumar@habana.ai>
---

diff --git a/package/zephyr/zephyr.mk b/package/zephyr/zephyr.mk
index 5b5e165..b3590a5 100644
--- a/package/zephyr/zephyr.mk
+++ b/package/zephyr/zephyr.mk
@@ -38,6 +38,7 @@
 ZEPHYR_CONF_OPTS_PID += -DBOARD=hl_$(DEVICE_NAME)_pid_$(HL_TARGET)
 ZEPHYR_CONF_OPTS_PID += -DDEVICE=$(DEVICE_NAME)
 ZEPHYR_CONF_OPTS_PID += $(dir $(BR2_EXTERNAL_HABANALABS_PATH))/arcpid
+ZEPHYR_CONF_OPTS_PID += -DZEPHYR_MODULES=$(dir $(BR2_EXTERNAL_HABANALABS_PATH))/zephyr-modules/boot_info
 ZEPHYR_CONFIGURE_CMDS += && $(ZEPHYR_CONF_ENV) $(HOST_DIR)/bin/cmake $(ZEPHYR_CONF_OPTS) $(ZEPHYR_CONF_OPTS_PID) -B $(ZEPHYR_PID_DIR)
 ZEPHYR_BUILD_CMDS += && $(TARGET_MAKE_ENV) $(MAKE) -C $(@D)/$(ZEPHYR_PID_DIR)
 ZEPHYR_INSTALL_TARGET_CMDS += && $(INSTALL) -D -m 0644 $(@D)/$(ZEPHYR_PID_DIR)/zephyr/zephyr.elf $(BINARIES_DIR)/zephyr_pid.elf
